### PR TITLE
Converted cash shop database to YAML

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: None

* **Server Mode**: Both

* **Description of Pull Request**: 
Converts the cash shop database to YAML.
Includes a converter for CSV.

### CODE REVIEW

File: src/map/cashshop.cpp [hunk 1]
```diff
@@ -15,144 +15,122 @@
...
-			}
+		if( !this->asUInt32( it, "Price", price ) ){
+			return 0;
+		}
 
-			if( !cashshop_parse_dbrow( str, 3, lines ) ) {
-				ShowError("cashshop_read_db_sql: Cannot process table '%s' at line '%d', skipping...\n", cash_db_name[fi], lines);
-				continue;
-			}
+		cash_item->price = price;
```
[the remark this answers is not in the document]
Comment (aleos89) [reply src/map/cashshop.cpp:98]: Agreed.

**Review by aleos89 — APPROVED** [review]

**Comment by Pokye** [comment]
Could you update with the current commit of rA please <3

**Comment by Lemongrass3110** [comment]
> Could you update with the current commit of rA please <3

@Pokye done.

File: src/tool/csv2yaml.cpp [hunk 2]
```diff
@@ -540,6 +540,13 @@ bool Csv2YamlTool::initialize( int argc, char* argv[] ){
 		return false;
 	}
 
+
```
Comment (aleos89) [inline src/tool/csv2yaml.cpp:543]:
```suggestion
```

**Review by aleos89 — APPROVED** [review]
Looks great!

**Pull request merged**